### ISSUE TITLE
Updates don't stick

### ISSUE THREAD

**Issue by Sqwrly** [MarshallOfSound/Google-Play-Music-Desktop-Player-UNOFFICIAL-#1883]
I started using GPMDP at version 4.0.1. Every time I start it up it asks me to update to 4.0.2. I say ok and it restarts in 4.0.2. But if I then close it and reopen it, it's back to 4.0.1 and will tell me to update again.

**Comment by MarshallOfSound** [comment]
Which OS?

**Comment by Sqwrly** [comment]
Sorry, should have known better. I'm running Windows 10 Pro x64 with the latest updates.

**Comment by MarshallOfSound** [comment]
Sometimes anti-virus gets in the way on Windows so I would suggest just installing the new version manually 👍

**Comment by Sqwrly** [comment]
I don't have any anti virus installed and Windows Defender is disabled in group policy.

**Comment by Elyviere** [comment]
I've been having this same issue for a while, also on Windows 10 64-bit. It currently also applies to the 4.0.3 version, and still reverts to 4.0.1. Here's what I've tried:

- Run as administrator
- Add as exception in antivirus
- Disable antivirus
- Change permissions to allow everything full control over the file

No combination of the above has resolved the problem.

**Comment by Sqwrly** [comment]
So I just figured this out about 10 minutes ago. When I originally downloaded this I just ran the exe that downloaded and it opened the player. It didn't seem to tell me that I had installed anything and I didn't see it in my start menu or any 'program files' directories. Apparently the downloaded exe is actually an installer that installs and runs that version(in our case 4.0.1). It then sees the version is out of date and updates. So what I did to find it was once I had the latest version running I opened task manager and selected "open file location" for GPMDP and it took me to the following directory: 'C:\Users\MYUSERNAME\AppData\Local\GPMDP_3\app-4.0.2'

After finding this I double checked my start menu to make sure there weren't any shortcuts, there weren't. But then I checked my desktop which I have all icons hidden on(I always have many full screen windows open, desktop is useless to me) and low and behold there was a shortcut to GPMDP there that starts up the latest version based on that above directory.
